### PR TITLE
tests: Add docker --cap-add and --cap-drop test.

### LINKED ISSUE #756: Add test to check capabilities
Test with docker ```--cap-add``` and ```--cap-drop```

### PR DESCRIPTION
**Pull request by GabyCT**
This test adds or drops different capabilities.

Fixes #756

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

File: docker.go [hunk 1]
```diff
@@ -36,6 +36,9 @@ const (
 
 	// DebianImage is the debian image
 	DebianImage = "debian"
+
+	// FedoraImage is the fedora image
+	FedoraImage = "fedora"
```
Comment (devimc) [inline docker.go]: please add this image here https://github.com/clearcontainers/tests/blob/master/integration/docker/main_test.go#L44

Comment (GabyCT) [reply docker.go]: @devimc changes applied

File: integration/docker/cap_test.go [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+	AfterEach(func() {
+		Expect(ExistDockerContainer(id)).NotTo(BeTrue())
+		Expect(ExistDockerContainer(anotherId)).NotTo(BeTrue())
+	})
+
+	DescribeTable("drop and add capabilities",
+		func(selectOption string) {
+			Skip("Issue https://github.com/clearcontainers/agent/issues/181") 
+			args = []string{"--name", id, "--rm", "--cap-drop", selectOption, FedoraImage, "sh", "-c", "capsh --print"}
```
Comment (devimc) [inline integration/docker/cap_test.go]: ```sh -c``` is not needed, you can run ```capsh``` directly

Comment (GabyCT) [reply integration/docker/cap_test.go]: @devimc changes applied

**Comment by devimc** [comment]
lgtm

**Pull request merged**